app/portainer/interceptors/redirectStatusInterceptor.js

```diff
@@ -0,0 +1,24 @@
+angular.module('portainer.app').factory('RedirectStatusInterceptor', [
```

Comment:
please refer to style guide https://portainer.atlassian.net/wiki/spaces/TECH/pages/45023521/Style+Guide

1. don't use dependencies array `[$q, $state,...]`, just add `/* @ngInject */ before the function declaration to automatically generate the array
2. remove 'use strict'
3. use es6+ features (const, Promise)

Reply:
4. do the following:
```
return {
  responseError = responseErrorInterceptor
}
```

instead of 
```
 var interceptor = {};

    interceptor.responseError = responseErrorInterceptor;

return interceptor
```

Reply:
see my comment on sharing this interceptor with axios. this should be defined as a regular function and not an angular factory

Reply:
fixed